### PR TITLE
update repo url references

### PR DESCRIPTION
**Pull request by tonypls**
## Description

We have added the "s" to the repos, updating all the links I could find that refer to the old repo urls

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 # Electricity Maps
 
-[![Slack Status](https://slack.electricitymaps.com/badge.svg)](https://slack.electricitymaps.com) [![CircleCI](https://circleci.com/gh/electricitymap/electricitymap-contrib.svg?style=shield)](https://circleci.com/gh/electricitymap/electricitymap-contrib) [![Twitter Follow](https://img.shields.io/twitter/follow/ElectricityMaps.svg?style=social&label=Follow)](https://twitter.com/ElectricityMaps)
+[![Slack Status](https://slack.electricitymaps.com/badge.svg)](https://slack.electricitymaps.com) [![CircleCI](https://circleci.com/gh/electricityMaps/electricitymaps-contrib.svg?style=shield)](https://circleci.com/gh/electricityMaps/electricitymaps-contrib) [![Twitter Follow](https://img.shields.io/twitter/follow/ElectricityMaps.svg?style=social&label=Follow)](https://twitter.com/ElectricityMaps)
```
Comment (VIKTORVAV99) [inline README.md:3]:
```suggestion
[![Slack Status](https://slack.electricitymaps.com/badge.svg)](https://slack.electricitymaps.com) [![CircleCI](https://circleci.com/gh/electricityMaps/electricitymaps-contrib.svg?style=shield)](https://circleci.com/gh/electricitymaps/electricitymaps-contrib) [![Twitter Follow](https://img.shields.io/twitter/follow/ElectricityMaps.svg?style=social&label=Follow)](https://twitter.com/ElectricityMaps)
```

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Looks like CircleCI don't like the capitalization (it works fine in the github links).

Besides that it looks good to me.

File: README.md (hunk 1 above)
Comment (tonypls) [reply README.md]: nice catch :)

**Review by VIKTORVAV99 — APPROVED** [review]

**Comment by VIKTORVAV99** [comment]
That cypress test that is failing is one of the flaky ones with a `.click()` method.

But why are we adding the readme to prettierignore?
(it's not really important either way but we might as well just ignore all `.md` files if we want to ignore the "text" files.)

**Comment by tonypls** [comment]
Ignored the readme because it is custom formatted based on how we want it to look. If you let prettier format it then it escapes any "*", formats JSON and removes padding we have to make it more readable.

**Comment by VIKTORVAV99** [comment]
> Ignored the readme because it is custom formatted based on how we want it to look. If you let prettier format it then it escapes any "*", formats JSON and removes padding we have to make it more readable. 

Hmm I thought prettier only removed excess characters (characters with no style impact) and formatted code blocks in markdown but I could be wrong.

I'll look into it though.

**Comment by tonypls** [comment]
Ah okay, must've been ESLint on my local machine doing the weird formatting.

**Pull request merged**